src/Form/EncryptionConfigurationForm.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/**
+ * @file
+ * Contains Drupal\encrypt\Form\EncryptionConfigurationForm.
+ */
+
+namespace Drupal\encrypt\Form;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Entity\EntityForm;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\encrypt\EncryptService;
+use Drupal\key\KeyManager;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class EncryptionConfigurationForm.
+ *
+ * @package Drupal\encrypt\Form
+ */
+class EncryptionConfigurationForm extends EntityForm {
+
+  /**
+   * KeyManager definition.
+   *
+   * @var \Drupal\key\KeyManager
+   */
+  protected $key_manager;
```

Comment:
Class variables should be camel case. These should be renamed to $keyManager and $encryptService.